Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -115,6 +115,11 @@ define([
      * @param {GetFeatureInfoFormat[]} [options.getFeatureInfoFormats] The formats in which to get feature information at a
      *                                 specific location when {@see UrlTemplateImageryProvider#pickFeatures} is invoked.  If this
      *                                 parameter is not specified, feature picking is disabled.
+     * @param {Boolean} [options.enablePickFeatures=true] If true, {@link UrlTemplateImageryProvider#pickFeatures} will
+     *        make ajax calls to options.pickFeaturesUrl for the features included in the response.  If false,
+     *        {@link UrlTemplateImageryProvider#pickFeatures} will immediately return undefined (indicating no pickable
+     *        features) without communicating with the server.  Set this property to false if you know your WMS server
```

Comment:
Shouldn't mention WMS here.  It's also worthwhile to mention that this parameter is ignored if `options.pickFeaturesUrl` is undefined.